jujugui/static/gui/src/app/components/entity-details/header/header.js

```diff
@@ -257,6 +257,38 @@ YUI.add('entity-header', function() {
     },
 
     /**
+     When the latest version link is click, go to that version.
+   */
+    _handleRevisionClick: function(e) {
+      e.stopPropagation();
+      this.props.changeState({
+        search: null,
+        store: this.props.entityModel.get('latest_revision').url
+      });
+    },
+
+    /**
+      Generates the list item to link to the latest version
+      or display the latest version
+   */
+    _generateLatestRevision: function() {
+      const latest_revision = this.props.entityModel.get('latest_revision');
+      const revision_id = this.props.entityModel.get('revision_id');
+
+      if (latest_revision.id !== revision_id) {
+        return <li>
```

Comment:
Any time JSX spans multiple lines, it's generally surrounded with parens and on its own line:

~~~~
return (
  <li>
    ...
  </li>
);
~~~~